<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_33">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="luxLabel_2">
           <property name="text">
            <string>valeur en Lux :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLCDNumber" name="lcdLuxmeter"/>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comBox"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="comButton">
           <property name="text">
            <string>changer port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="comLabel_2">
           <property name="text">
            <string>Port de communication</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_34">
         <item>
          <widget class="QTabWidget" name="captureTab_2">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>échantillonnage</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_35">
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_27">
                <item>
                 <widget class="QLabel" name="TimerLabel_2">
                  <property name="text">
                   <string>Temps de capture :(sec)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="inter_capture">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>3600</number>
                  </property>
                  <property name="singleStep">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_28">
                <item>
                 <widget class="QLabel" name="intervalleLabel_2">
                  <property name="text">
                   <string>Intervalle de capture :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="inter_capture_2">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>60</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="timerButton">
               <property name="text">
                <string>capture</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>instantanée</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <item>
              <widget class="QSplitter" name="splitter_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QSplitter" name="splitter_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QPushButton" name="inst_capture">
                 <property name="text">
                  <string>capture</string>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
